txnprovider/txpool/pool_txn_parser.go

```diff
@@ -493,11 +494,13 @@ func (ctx *TxnParseContext) parseTransactionBody(payload []byte, pos, p0 int, sl
 			if err != nil {
 				return 0, fmt.Errorf("%w: authorization nonce: %s", ErrParseTxn, err) //nolint
 			}
+			rawEnd := p2
 			p2, _, err = parseSignature(payload, p2, false /* legacy */, nil /* cfgChainId */, &sig)
 			if err != nil {
 				return 0, fmt.Errorf("%w: authorization signature: %s", ErrParseTxn, err) //nolint
 			}
 			slot.Authorizations = append(slot.Authorizations, sig)
+			slot.AuthRaw = append(slot.AuthRaw, payload[rawStart:rawEnd+1])
```

Comment:
Is this ok? Any better way to do this?

Also need another check on if we really need the +1 here.

Reply:
Better use `libcommon.CopyBytes`. Please also double check whether +1 is needed and add a test for it.